app/src/renderer/components/common/TmSessionHardware.vue

```diff
@@ -50,13 +44,15 @@ export default {
     setStatus(value) {
       this.status = value
     },
-    onSubmit() {
-      this.$store.commit(`setModalSession`, false)
-      this.$store.commit(`notify`, {
-        title: `Welcome back!`,
-        body: `You are now signed in to your Cosmos account.`
-      })
-      this.$store.dispatch(`signIn`, { password: this.fields.signInPassword })
+    async connectLedger() {
```

Comment:
@faboweb @fedekunze i don't think we should be signing people in with their ledgers. it is unnecessary, provides more room for error, and will lead people to think that maybe they won't need to sign individual transactions.

Reply:
See comment on https://github.com/cosmos/voyager/issues/1917

Reply:
This was done in order to get the address in order to query balances, delegations, etc. 

Imo we should still provide the option to sign in with the wallet. Later we can add support for signing with ledger without logging in when #1718 is done. Adding that now feature will throw errors everywhere